tools/walletextension/rpcapi/transaction_api.go

```diff
@@ -0,0 +1,149 @@
+package rpcapi
+
+//goland:noinspection ALL
```

Comment:
How comes you had to put this goland disable in in some places?

Reply:
will check.